### PR TITLE
e2e - save logs tests

### PR DESCRIPTION
**Pull request by tanyatrayanava**

### CODE REVIEW

File: tests/e2e/tests/critical-path/monitor/save-commands.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    .page(commonUrl)
+    .beforeEach(async() => {
+        await acceptLicenseTermsAndAddDatabase(ossStandaloneConfig, ossStandaloneConfig.databaseName);
+    })
+    .afterEach(async() => {
+        //Delete database
+        await deleteDatabase(ossStandaloneConfig.databaseName);
+    });
+test
+    .meta({ rte: rte.standalone })('Verify that user can see a tooltip and toggle that allows to save Profiler log or not in the Profiler', async t => {
```
Comment (ofersteinberg) [inline tests/e2e/tests/critical-path/monitor/save-commands.e2e.ts:27]: Please set the `meta` on the fixture level if all are the same

File: tests/e2e/tests/regression/monitor/save-commands.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    .page(commonUrl)
+    .beforeEach(async() => {
+        await acceptLicenseTermsAndAddDatabase(ossStandaloneConfig, ossStandaloneConfig.databaseName);
+    })
+    .afterEach(async() => {
+        //Delete database
+        await deleteDatabase(ossStandaloneConfig.databaseName);
+    });
+test
+    .meta({ rte: rte.standalone })('Verify that when clicks on “Reset Profiler” button he brought back to Profiler home screen', async t => {
```
Comment (ofersteinberg) [inline tests/e2e/tests/regression/monitor/save-commands.e2e.ts:25]: Same comment

File: tests/e2e/tests/regression/monitor/save-commands.e2e.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        await t.click(monitorPage.runMonitorToggle);
+        await t.click(monitorPage.resetProfilerButton);
+        //Check the screen
+        await t.expect(monitorPage.monitorNotStartedElement.visible).ok('The Profiler home screen appears');
+        await t.click(monitorPage.closeMonitor);
+        //Start Monitor with Save logs
+        await monitorPage.startMonitorWithSaveLog();
+        //Reset profiler
+        await t.click(monitorPage.runMonitorToggle);
+        await t.click(monitorPage.resetProfilerButton);
```
Comment (ofersteinberg) [inline tests/e2e/tests/regression/monitor/save-commands.e2e.ts:40]: You could create function within the test of this action

File: tests/e2e/tests/critical-path/monitor/save-commands.e2e.ts (hunk 1 above)
Comment (tanyatrayanava) [reply tests/e2e/tests/critical-path/monitor/save-commands.e2e.ts:27]: done

File: tests/e2e/tests/regression/monitor/save-commands.e2e.ts (hunk 2 above)
Comment (tanyatrayanava) [reply tests/e2e/tests/regression/monitor/save-commands.e2e.ts]: done

File: tests/e2e/tests/critical-path/monitor/save-commands.e2e.ts (hunk 1 above)
Comment (ofersteinberg) [reply tests/e2e/tests/critical-path/monitor/save-commands.e2e.ts:27]: So do you use with both fixture meta 'regression' and this one?

Comment (tanyatrayanava) [reply tests/e2e/tests/critical-path/monitor/save-commands.e2e.ts:27]: yes

**Pull request merged**